src/IO/readFloatText.h

```diff
@@ -590,7 +590,19 @@ template <typename T> bool tryReadFloatTextSimple(T & x, ReadBuffer & in) { retu
 
 /// Implementation that is selected as default.
 
-template <typename T> void readFloatText(T & x, ReadBuffer & in) { readFloatTextFast(x, in); }
-template <typename T> bool tryReadFloatText(T & x, ReadBuffer & in) { return tryReadFloatTextFast(x, in); }
+template <bool precise_float_parsing, typename T> void readFloatText(T & x, ReadBuffer & in)
```

Comment:
No need for template parameters, just call the right function directly by the caller.

Reply:
Done